tests/infer/mcmc/test_hmc.py

```diff
@@ -171,7 +171,7 @@ def model(data):
     assert_equal(rmse(true_coefs, beta_posterior.mean).item(), 0.0, prec=0.1)
 
 
-def test_bernoulli_beta():
+def test_beta_bernoulli():
```

Comment:
Ha ha this has always annoyed me, thanks for swapping 😆

Reply:
Same here, it's been on my mind for some time. :)